include/bluetooth/services/fast_pair.h

```diff
@@ -46,7 +46,20 @@ size_t bt_fast_pair_adv_data_size(bool fp_discoverable);
 int bt_fast_pair_adv_data_fill(struct bt_data *adv_data, uint8_t *buf, size_t buf_size,
 			       bool fp_discoverable);
 
-/** Enable or disable Fast Pair pairing mode
+/** Show or hide UI indication on Fast Pair Seeker device.
+ *
+ * Device must be Fast Pair not discoverable and the Account Key List must be not empty for
+ * show/hide UI indication option to be used.
+ *
+ * It's user responsibility to make sure that proper UI indication option is enabled before
+ * advertising is started. By default, show UI indication option is enabled.
+ *
+ * @param[in] fp_show_ui_indication Boolean indicating if UI indication are to be shown on Fast Pair
+ *                                  Seeker device.
+ */
+void bt_fast_pair_show_ui_indication(bool fp_show_ui_indication);
```

Comment:
Consider defining ```struct bt_fast_pair_adv_data_config``` structure to define the configuration of the Fast Pair advertising data. You can use it as a fourth parameter of the ```bt_fast_pair_adv_data_fill``` function. In this case, this API function won't be necessary.

I believe that my API suggestion gives the user more flexibility to encode Fast Pair advertising buffers from multiple contexts (if there is a need for this) and manage configuration memory on their own

Reply:
I wanted to do it the same way as `bt_fast_pair_set_pairing_mode` sets pairing mode in `fp_keys` module to be consistent with already existent API.

Reply:
The pairing mode configuration needs to persist after the bt_fast_pair_set_pairing_mode so that the fp_keys module can work properly. I believe that this UI indication is just used as some data encoded in the advertising payload and can be passed when advertising data encoding is being requested